mainnet-assets.json

```diff
@@ -397,5 +397,15 @@
     "subId": "0x36470f078fb1e8d5922c44b01ffa7190258827f7a6de97edea790ca182503d72",
     "assetId": "0x4fc8ac9f101df07e2c2dec4a53c8c42c439bdbe5e36ea2d863a61ff60afafc30",
     "isVerified": true
+  },
+
+  {
+    "name": "Psycho Ducky",
+    "symbol": "PSYCHO",
+    "decimals": 9,
```

Comment:
icon.default is a required field (haven't had time to add docs & validation yet)